bin/fuel-core/src/cli.rs

```diff
@@ -127,3 +129,417 @@ pub async fn run_cli() -> anyhow::Result<()> {
         }
     }
 }
+
+#[cfg(any(feature = "rocksdb", feature = "rocksdb-production"))]
+#[cfg(test)]
+mod tests {
```

Comment:
These are some stellar tests. I've never really tested against the cli commands directly before, but I like how this makes very living docs of how it works :muscle:

Reply:
Props to @segfault-magnet